app/src/main/java/com/infomaniak/swisstransfer/ui/screen/newtransfer/importfiles/components/TransferAdvancedSetting.kt

```diff
@@ -70,9 +70,9 @@ fun TransferAdvancedSetting(
 }
 
 @Composable
-private fun SettingValue(selectedSetting: () -> TransferAdvancedOptionsEnum) {
+private fun SettingValue(selectedSetting: () -> SettingOption?) {
     Text(
-        text = selectedSetting().title(),
+        text = selectedSetting()?.title?.invoke() ?: "",
```

Comment:
@sirambd was talking about how we should have whole different empty screen for a split second while waiting for the realm data and only when we received the data can we show the whole screen and at this point all these values would be non-null. What do we do about it?

Reply:
That feels overkill (at least for a beta)

Reply:
But a thing that is certain is that it should not be in this PR if we chose to add this